src/dev_bot.py

```diff
@@ -8,24 +8,34 @@
 
 import logging
 
-from aiogram import types
+from aiogram import types, Bot
 
 from bot_behaviors.random_responses import random_response
 from bot_behaviors.experience import Experience
 from database.database import Database
-
+from decouple import config
+bot = Bot(token= config('API_TOKEN'))
 
 class DevBot:
     def __init__(self, dispatcher):
         self.dispatcher = dispatcher
+        self.bot = bot
         self.database = Database()
         self.experience = Experience(self.database)
         self.run_bot()
 
     def run_bot(self):
+        async def expname(chatid, userid):
```

Comment:
Essa função não está no lugar correto e não tem um nome muito claro, acho que inclusive seria melhor remover ela, e fazer a chamada diretamente ali embaixo na função exp().

Reply:
Concordo com a alteração da nomenclatura, mas o local é esse mesmo!
Ele precisa ser uma função por dois motivos:
- Primeiro lugar, precisaremos dela no futuro (pense sempre em novas utilizações ao fazer uma função)
- Segundo lugar, o método que estou usando precisar de argumentos e, os argumentos que estão inserindo está sendo em loop (for), desta forma prejudicará o processamento fora que as async questiona o tempo de requisição

Nota: aonde ele está, está ok!

Reply:
Enviei correção pelo commit: [c2e5348](https://github.com/GrupoDevelopers/DevBot/pull/21/commits/c2e53482b85f513dd16488042a875f94b55d9773)